tests/unit/test_run.py

```diff
@@ -2371,6 +2387,66 @@ def test_interpolate_single_constant():
         df.interpolate(target)
 
 
+@pytest.mark.parametrize(
+    "years,values",
+    [
+        ([2000, 2001, 2002, 2003], [np.nan, 2.0, 3.0, 4.0]),
+        # We assume Jan 1 values so the following doesn't cover a leap year
+        # [2001, 2002, 2003, 2004],[1.0, 2.0, 3.0, np.nan]),
```

Comment:
```suggestion
        pytest.param([2001, 2002, 2003, 2004],[1.0, 2.0, 3.0, np.nan]), mark=pytest.xfail(reason="Doesn't cover a leap year"))
```

Something like that should work I think and might be clearer than having this commented out?